va/nfv/nfv-ovs-dpdk-sriov-hci/README.md

```diff
@@ -0,0 +1,11 @@
+# VA: OvS DPDK
+
+If you are looking for information on how to deploy the OvS DPDK & SRIOV VA, then
+please see the
+[OvS DPDK & SRIOV README in the examples directory](../../../examples/va/nfv/ovs-dpdk-sriov/README.md).
+
+This directory, `architecture/va/nfv/ovs-dpdk-sriov/`, exists so that the
+[kustomization.yaml](../../../examples/va/nfv/ovs-dpdk-sriov/kustomization.yaml)
+in the examples directory for the OvS DPDK & SRIOV VA, reference it by path as a
```

Comment:
```suggestion
# VA: OvS DPDK & SRIOV HCI

If you are looking for information on how to deploy the OvS DPDK & SRIOV HCI VA, then
please see the
[OvS DPDK & SRIOV HCI README in the examples directory](../../../examples/va/nfv/nfv-ovs-dpdk-sriov-hci/README.md).

This directory, `architecture/va/nfv/nfv-ovs-dpdk-sriov-hci/`, exists so that the
[kustomization.yaml](../../../examples/va/nfv/nfv-ovs-dpdk-sriov-hci/kustomization.yaml)
in the examples directory for the OvS DPDK & SRIOV HCI VA, reference it by path as a
```